pkg/operator/starter.go

```diff
@@ -246,3 +252,39 @@ func isCustomCABundleUsed(cloudConfigLister corev1listers.ConfigMapNamespaceList
 	_, exists := cloudConfigCM.Data[caBundleKey]
 	return exists, nil
 }
+
+// withCustomTags add tags from Infrastructure.Status.PlatformStatus.AWS.ResourceTags to the driver command line as
+//  --extra-tags=<key1>=<value1>,<key2>=<value2>,...
+func withCustomTags(infraLister v1.InfrastructureLister) csidrivercontrollerservicecontroller.DeploymentHookFunc {
+	return func(spec *opv1.OperatorSpec, deployment *appsv1.Deployment) error {
+		infra, err := infraLister.Get(infrastructureName)
+		if err != nil {
+			return err
+		}
+		if infra.Status.PlatformStatus == nil || infra.Status.PlatformStatus.AWS == nil {
+			return nil
```

Comment:
Is it possible to add some `logs` ? .. 
I can see the same func could return `err` as nil in many cases, logs could eventually help I guess